DeviceTests/Caboodle.DeviceTests.Shared/Battery_Tests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Caboodle;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Caboodle.DeviceTests
+{
+    public class Battery_Tests
+    {
+        [Fact]
+        public void Charge_Level() =>
+            Assert.Equal(1.0, Battery.ChargeLevel);
+
+        [Fact]
+        public void Charge_State() =>
+            Assert.Equal(BatteryState.Full, Battery.State);
+
+        [Fact]
+        public void Charge_Power() =>
+            Assert.Equal(BatteryPowerSource.Ac, Battery.PowerSource);
```

Comment:
Will these tests pass on an disconnected device with a low battery? 😄

Reply:
I don't think so :( My assumption here is we are testing on emulators? how  should we handle this? Anything but known?

Reply:
yeah... you could try checking if level > 0... It probably will always be on ac/usb for tests as it has to be connected to deploy/run the tests.